docs/self-managed/platform-deployment/overview.md

```diff
@@ -19,8 +19,9 @@ Camunda 8 includes the following components:
 - Optimize (requiring Elasticsearch and Identity)
 - Identity (requiring Keycloak)
 - Web Modeler (requiring Identity, Keycloak, and PostgreSQL) [<span class="badge badge--enterprise-only">Enterprise only</span>](../../../reference/licenses/#web-modeler)
+- Console [<span class="badge badge--enterprise-only">Enterprise only</span>](../../../reference/licenses/#web-modeler)
```

Comment:
link to license of console here?

Reply:
@felix-mueller Who is responsible for the license?

Reply:
@urbanisierung legal is responsible for it, for now we would use enterprise-only similar to web modeler. 
we need to make sure we have the license and license header as part of the source code. unsure if we have this already?

Reply:
done